lib/logthrdestdrv.c

```diff
@@ -34,7 +34,7 @@ log_threaded_dest_driver_set_max_retries(LogDriver *s, gint max_retries)
 {
   LogThreadedDestDriver *self = (LogThreadedDestDriver *)s;
 
-  self->retries.max = max_retries;
+  self->retries_max = max_retries;
```

Comment:
Personally I like the coupling of max and counter fields. Especially if the `retries_over` were part of the same structure.

Reply:
I also like it, but only when we are not using it for prefixing and are actually grouping related fields. The worst offender was format.stats_instance, with a single member, but we also had messages.retry_over, again as a single member.

I might have gone the other side too much by also getting rid off retries, but I did that because the original structure was completely mixed and has been confusing. I did keep "worker" though, I even added a few members.

As I look at the structure, I would still probably  not use "retries" as a struct name, rather "batch" or "batching", which could contain:
* size
* retry_max
* retry_attempts

Again to cluster not just instead of prefixing, but along functionality. Do you want me to do that?